client/app/src/js/controllers/screenshot.js

```diff
@@ -4,7 +4,7 @@
  * Handles display of a single screenshot.
  */
 
-function ScreenshotController(Screenshot, User) {
+function ScreenshotController(Screenshot, User, screenshot) {
 
   this.screenshot = {};
 
```

Comment:
Is the `this.getScreenshot` function below redundant, now that we're injecting the screenshot from `resolve`?